### ISSUE TITLE
Disable kicking users if the group does not allow to send messages

### ISSUE THREAD

**Issue by mrkct** [1inch/shieldy#167]
Recently the admin of a telegram group chat I'm in decided pin a message redirecting people to a new group and to disallow everyone to send messages in the old group.  

Some people still join the old group via link, but they get kicked by Shieldy since they cannot send a message. It would be nice if Shieldy disabled this feature if it notices that the user cannot send messages

**Comment by backmeupplz** [comment]
it simply should not be used in groups with disabled message sending

**Comment by mrkct** [comment]
You're right, it's just that I imagine people add the bot and forget about it. It would be a nice quality of life feature. I don't know how hard it is to implement

**Comment by Kwaskoff** [comment]
and don't kick users when they didn't pass captcha - only restricted messaging
